web/themes/custom/novel/templates/components/list-teaser.html.twig

```diff
@@ -0,0 +1,49 @@
+{%
+  set classes = [
+  'event-list-item',
+  'arrow__hover--right-small',
+]
+%}
+
+<a {{ attributes.addClass(classes).setAttribute('href', url) }}>
+  <div class="event-list-item__image-container">
+    {{ image }}
+  </div>
+  <div class="event-list-item__content">
```

Comment:
Please reconsider naming here.

This is named as generic list container but the class names are prefixed with `event-list`. That does not seem like a good idea. Does danskernesdigitalebibliotek/dpl-design-system#527 need more refactoring?